<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaginationWidget</class>
 <widget class="QWidget" name="PaginationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="buttonPrev">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/Resources/left-arrow.svg</normaloff>:/Resources/left-arrow.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pageButtonsWidget" native="true"/>
   </item>
   <item>
    <widget class="QToolButton" name="buttonNext">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/Resources/right-arrow.svg</normaloff>:/Resources/right-arrow.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="buttonJumpPrev">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/Resources/left-arrow.svg</normaloff>:/Resources/left-arrow.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditJumpTo">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonJumpNext">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/Resources/right-arrow.svg</normaloff>:/Resources/right-arrow.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxItemsPerPage"/>
   </item>
   <item>
    <widget class="QLabel" name="labelItemsPerPage">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
